pandas/tests/io/json/test_pandas.py

```diff
@@ -226,6 +226,11 @@ def test_roundtrip_empty(self, orient, convert_axes, numpy, empty_frame):
     @pytest.mark.parametrize("numpy", [True, False])
     def test_roundtrip_timestamp(self, orient, convert_axes, numpy, datetime_frame):
         # TODO: improve coverage with date_format parameter
+
+        # freq doesnt round-trip
```

Comment:
why don’t u just change this in the fixture? tsframe

Reply:
updated per suggestion

Reply:
gentle ping; plenty more of these before were done